libs/ui/components/contents/organization-menu/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { VStack } from '@chakra-ui/react';
+import { IOrganizationMenuProps } from './props';
+import { OrganizationMenuItem } from './organization-menu-item';
+export default function OrganizationMenu<T>({
+    options,
+    value,
+    onChange,
+}: IOrganizationMenuProps<T>) {
+    // Functions
+    const renderOptions = (): React.ReactNode => {
+        return options.map((obj) => (
+            <OrganizationMenuItem
+                title={obj.title}
+                description={obj.description}
+                isActive={value === obj.value}
+                imageUrl={obj.imageUrl}
+                onSelect={() => onChange?.(obj.value as T)}
```

Comment:
```suggestion
                onSelect={() => onChange?.(obj.value)}
```
No need to specify T, if yes there is an issue in the props

Reply:
solve the problem was one props have value?: T and another have value : T